Options/WeakAurasOptions.lua

```diff
@@ -4799,6 +4786,31 @@ function WeakAuras.ReloadTriggerOptions(data)
         end
       end
     },
+    activeTriggerMode = {
+      type = "select",
+      name = L["Use information from trigger"],
```

Comment:
This might need to be worded differently, what it does is use this triggers icon and name, right? So maybe `Use this trigger's data for display`?